backend/.eslintrc.json

```diff
@@ -15,7 +15,7 @@
             2,
             "always"
         ],
-        "linebreak-style": 1,
+        "linebreak-style": 0,
```

Comment:
I am just wondering why did you turn off linebreak-style rule?

Reply:
On Vs code you can do this : 
![image](https://user-images.githubusercontent.com/45897337/209848304-8280ff96-0f20-4258-b7b0-199a38a9e465.png)

Reply:
Yes , I enabled that settings in vs code, Thanks for the info. 
I changed that line-break rule for my convenience . Sorry, I forgot to restore it while committing, .

Should I make the changes in that line and make another commit?

Reply:
already did it. All looks good to me, I will add env variable to the server, and merge your PR 🎉